packages/core/src/App/Containers/Layout/header/dashboard-header.jsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { useHistory } from 'react-router-dom';
+import { Button, Text } from '@deriv/components';
+import { PlatformContext, redirectToLogin, redirectToSignUp, routes } from '@deriv/shared';
+import { getLanguage } from '@deriv/translations';
+
+const DashboardHeader = () => {
+    const history = useHistory();
+    const { is_deriv_crypto } = React.useContext(PlatformContext);
+
+    return (
+        <header className='dashboard-header'>
+            <div className='dashboard-header__menu-items'>
+                <div className='dashboard-header__menu-items-left'>
+                    <Text color='colored-background' size='m' onClick={() => history.push(routes.dashboard)}>
+                        Deriv
+                    </Text>
+                </div>
+                <div className='dashboard-header__menu-items-middle'>
+                    <Text color='colored-background' size='s' onClick={() => history.push(routes.explore)}>
+                        Explore
+                    </Text>
+                    <Text color='colored-background' size='s' onClick={() => history.push(routes.about_us)}>
+                        About us
+                    </Text>
+                    <Text color='colored-background' size='s' onClick={() => history.push(routes.resources)}>
```

Comment:
small ('s') is the default value for size in **Text** so you can skip it.

Reply:
This is a temporary page so let's not worry about styling.